### PR TITLE
ir: Do not configure network settings

### PR DESCRIPTION
**Pull request by cthulhu-rider**

### CODE REVIEW

**Comment by cthulhu-rider** [comment]
deployment works with https://github.com/nspcc-dev/neofs-dev-env/pull/282 but IR does not become healthy, will take a look

File: pkg/morph/deploy/deploy.go [hunk 1]
```diff
@@ -462,7 +440,53 @@ func Deploy(ctx context.Context, prm Prm) error {
 
 	prm.Logger.Info("Audit contract successfully synchronized", zap.Stringer("address", auditContractAddress))
```
Comment (roman-khimov) [inline pkg/morph/deploy/deploy.go:441]: You still deploy audit the first (from regular non-NNS contracts), but it's named `02-audit` now. Probably a rename or reorder is needed.

**Review by roman-khimov — COMMENTED** [review]
OK otherwise

File: pkg/morph/deploy/deploy.go (hunk 1 above)
Comment (cthulhu-rider) [reply pkg/morph/deploy/deploy.go:441]: Audit contract is deployed after the Proxy one

Comment (roman-khimov) [reply pkg/morph/deploy/deploy.go:441]: OK, then the numbering here and the numbering in contracts don't match exactly. Not a big deal though.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**